includes/class.llms.install.php

```diff
@@ -448,6 +449,7 @@ private static function get_schema() {
   `status` varchar(15) DEFAULT '',
   `attempt` bigint(20) DEFAULT NULL,
   `grade` float DEFAULT NULL,
+  `can_be_resumed` tinyint(1) DEFAULT '0',
```

Comment:
Check that this will automatically get picked up as a new column without a migration.